core/src/main/java/hudson/cli/StandardCLIReturnCode.java

```diff
@@ -42,63 +48,74 @@
  * @since TODO
  */
 public enum StandardCLIReturnCode implements CLIReturnCode {
-	/**
-	 * Everything went ok
-	 * HTTP equivalent: 2xx
- 	 */
-	OK(0),
-	/**
-	 * Internal server error
-	 * HTTP equivalent: 500
- 	 */
-	UNKNOWN_ERROR_OCCURRED(1),
-	/**
-	 * Input cannot be decoded or are wrong
-	 * HTTP equivalent: 400
- 	 */
-	WRONG_CMD_PARAMETER(2),
-	/**
-	 * Wrong input arguments
-	 * HTTP equivalent: 400, 404
-	 * Example: job doesn't exist
- 	 */
-	ILLEGAL_ARGUMENT(3),
-	/**
-	 * Correct input but wrong state
-	 * HTTP equivalent: 400, 410
-	 * Example: build is already finished
-	 */
-	ILLEGAL_STATE(4),
-	/**
-	 * Can't continue due to an other (rare) issue
-	 */
-	ABORTED(5),
-	/**
-	 * User is authenticated but does not have sufficient permission
-	 * HTTP equivalent: 403
- 	 */
-	ACCESS_DENIED(6),
-	/**
-	 * Credentials sent but are invalid
-	 * HTTP equivalent: 401
- 	 */
-	BAD_CREDENTIALS(7),
+    /**
+     * Everything went ok
+     * HTTP equivalent: 2xx
+     */
+    OK(0, null),
```

Comment:
`null` or `"Completed Successfully"`?  making this non-null may make things easier and knowing something was "ok" is OK is it not?